data/free worlds reconciliation.txt

```diff
@@ -1633,14 +1635,65 @@ mission "FW Pug 3C"
 		dialog "You have reached <planet>, but part of your fleet is not here yet. Better depart and wait for them to arrive in this star system."
 
 
+mission "FW Pug 3C(Extra Ship)"
```

Comment:
Instead of duplicating the missions with the same conversations but different numbers of NPCs, you should have the escorts be a separate mission akin to FW Pug 4: Escorts.